build/cmd/zonedb/main.go

```diff
@@ -28,6 +28,7 @@ func main() {
 	listZones := flag.Bool("list", false, "list working zones")
 	listTags := flag.Bool("list-tags", false, "list tags in working zones")
 	listLocations := flag.Bool("list-locations", false, "list locations in working zones")
+	listMetros := flag.Bool("list-metros", false, "list metros in working zones")
```

Comment:
I think we should use `metro codes` `metroCodes` `MetroCodes` instead of bare word `metros` in this change set. What is the ISO (or other) standard these come from?